src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/VtableIndexStubGenerator.cs

```diff
@@ -292,44 +344,42 @@ private static IncrementalStubGenerationContext CalculateStubInformation(MethodD
             }
             else
             {
-                typeKeyType = ManagedTypeInfo.CreateTypeInfoForTypeSymbol(iUnmanagedInterfaceTypeInstantiation.TypeArguments[0]);
+                typeKeyType = ManagedTypeInfo.CreateTypeInfoForTypeSymbol(iUnmanagedInterfaceTypeInstantiation.TypeArguments[1]);
```

Comment:
Why "1"?

Reply:
The definition of `IUnmanagedInterfaceType<>` changed and now the type key is the second type argument instead of the first. I'll add a comment. I don't think it's worth trying to future proof by checking for the type parameter based on name as the type parameter list is going to be unchangable after we ship anyway.